README.md

```diff
@@ -44,19 +44,14 @@ _Note: some terminal commands may take a while to process, without offering feed
 
 1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder. You can do this by running `cp config/local{.sample,}.js`
 This will be the file that holds your S3 and SQS configurations.
-1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). Give your app a name and "Homepage URL" (`http://localhost:1337`), and use `http://localhost:1337/auth/github/callback` as the "Authorization callback url".
+1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). Give your app a name and "Homepage URL" (`http://localhost:1337`), and use `http://localhost:1337/auth/github2/callback` as the "Authorization callback url".
```

Comment:
This should be both urls is possible: `http://localhost:1337/auth/github/callback`, `http://localhost:1337/auth/github2/callback`

Reply:
Are you saying we should add both URLs to one application, or add two applications? The GitHub "Register a new OAuth application" only has a field for one callback URL. When I entered `http://localhost:1337/auth/github/callback` I was unable to authenticate in local Federalist using GitHub. When I changed it to `ttp://localhost:1337/auth/github2/callback` it fixed the problem.

Reply:
aaaahhhhhh, ok, maybe 2 applications makes more sense:
- localhost:1337/auth/github2/callback
- localhost:1338/auth/github/callback

Reply:
I went in to adjust the README with this and realized I don't know how it's supposed to be set up. It's easy enough to make two GitHub applications, but how is a developer supposed to edit `githuboptions` in `config/local.js` to represent both of them? `config/local.sample.js` only has keys `clientID` and `clientSecret`. Can you provide an example of a `local.js` configuration that binds to two GitHub OAuth apps?

Reply:
`local.sample.js` does this. there are `options`, `authorizationOptions`, and `externalOptions`, those are used in different Github `passport` "strategies" in the various `passport.js` configurations.

This is confusing and should be cleaned up post-SCR.

Reply:
All right. `passport.js` associates `authorizationOptions` with "github2" and `options` with "github". I suppose `authorizationOptions` is used at :1337 and `options` is used at :1338? Do you recommend I update `local.sample.js` to standardize two separate configurations, or should I leave README referring only to the business happening at :1337?

Reply:
just update the README for now, the work to clean this up will happen after the SCR.

Reply:
In that case I think my change to the "github2" callback for the single OAuth application is probably sufficient as it stands. Developers won't need to set the others unless they're working at :1338, and that port isn't documented at all in the README and can perhaps wait until after the SCR as well.